tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react",
-    "baseUrl": "./src"
+    "baseUrl": "./src",
+    "noFallthroughCasesInSwitch": true
```

Comment:
[note] This was added automatically after we upgraded `react-scripts` the first time we `npm start`'d